ivy/container/linear_algebra.py

```diff
@@ -2379,13 +2379,10 @@ def tensorsolve(
         x2: Union[ivy.Container, ivy.Array, ivy.NativeArray],
         /,
         *,
-        axes: Union[int, Tuple[List[int], List[int]]] = None,
+        axes: Optional[Union[int, Tuple[List[int], List[int]]]] = None,
     ) -> ivy.Container:
-        return self.tensorsolve(
-            self,
-            x2,
-            axes=axes,
-        )
+        # TODO: implement this function
+        pass
```

Comment:
Is there no `tensorsolve`?

Reply:
https://github.com/unifyai/ivy/blob/b98068a7e60162f83e1003b18c52314494addb91/ivy/functional/ivy/linear_algebra.py#L2141

Reply:
The old implementation was wrong, and I was scared this PR may go out of scope so I left it as a TODO :sweat_smile:

Reply:
I didn't think it through when I saw you deleted the function call tbh 😅 You can leave it as it is now, no need to add docstrings.